--- alignment.c-org	2011-02-11 10:50:37.000000000 -0600
+++ alignment.c	2011-11-16 11:42:10.000000000 -0600
@@ -522,7 +522,7 @@
       for (sj = si + 1; sj < nseqs; sj++) {
          m = seqlen_array[sj+1];
          if ( n == 0 || m == 0 ) {
-            bench_output[si*nseqs+sj] = (int) 1.0;
+             seq_output[si*nseqs+sj] = (int) 1.0;
          } else {
             int se1, se2, sb1, sb2, maxscore, seq1, seq2, g, gh;
             int displ[2*MAX_ALN_LENGTH+1];
@@ -649,7 +649,6 @@
 {
    int i,j;
    seq_output = (int *) malloc(sizeof(int)*nseqs*nseqs);
-   bench_output = (int *) malloc(sizeof(int)*nseqs*nseqs);
 
    for(i = 0; i<nseqs; i++)
       for(j = 0; j<nseqs; j++)
